Documentation/ProjectFile/prj/processes/process/THERMO_HYDRO_MECHANICS/t_reference_temperature.md

```diff
@@ -0,0 +1 @@
+Reference temperature for the thermal part of thermo-hydro-mechanical process which is normally same with initial temperature.
```

Comment:
Is the reference temperature really normally the same **as** the initial temperature?

Reply:
For many cases, it can be same with the initial temperature. But for sure, there are a lot of other cases. This sentence is deleted.